packages/fast-components/src/text-area/text-area.template.ts

```diff
@@ -0,0 +1,38 @@
+import { html, ref, when } from "@microsoft/fast-element";
+import { TextArea } from "./text-area";
+
+export const TextAreaTemplate = html<TextArea>`
+    ${when(
```

Comment:
This will work for initial render, but not if the child nodes change later. We've already discussed so we'll leave this as is for now. I've got a new feature coming, hopefully next week, that will enable us to do this in a more robust way.